torchci/scripts/td_heuristic_historical_edited_files.py

```diff
@@ -0,0 +1,61 @@
+import json
+from collections import defaultdict
+from typing import Dict
+
+from utils_td_heuristics import (
+    cache_json,
+    evaluate,
+    get_all_invoking_files,
+    get_filtered_failed_tests,
+    get_merge_bases_dict,
+    list_past_year_shas,
+    query_rockset,
+)
+
+CHANGED_FILES_QUERY = """
+select
+    sha,
+    changed_files
+from
+    commons.merge_bases
+where
+    ARRAY_CONTAINS(SPLIT(:shas, ','), sha)
+"""
+
+
+@cache_json
+def gen_correlation_dict() -> Dict[str, Dict[str, float]]:
+    shas = list_past_year_shas()
+
+    interval = 500
+    commits = []
+    for i in range(0, len(shas), interval):
+        commits.extend(
+            query_rockset(
+                CHANGED_FILES_QUERY,
+                params={"shas": ",".join(shas[i : i + interval])},
+                use_cache=True,
+            )
+        )
+
+    invoking_files = get_all_invoking_files()
+
+    d = defaultdict(lambda: defaultdict(float))
+    for commit in commits:
+        changed_files = commit["changed_files"]
+        test_files = [x[5:-3] for x in changed_files if x[5:-3] in invoking_files]
```

Comment:
What's special about this range of rows? Can you add a comment, or better yet replace these numbers with constants that help explain the significance of this range?  More specifically, why are the other rows excluded from this range?